src/components/student/ScheduleView.js

```diff
@@ -9,14 +14,130 @@ import React, {useState} from 'react';
 // issue a DELETE with URL /enrollment/{enrollmentId}
 
 const ScheduleView = (props) => {
-    
-   
-    return(
-        < > 
-            <h3>Not implemented</h3>
-        </ >
-    );
-
-}
+
+    const headers = ['SecNo', 'CourseId', 'SecId',  'Year', 'Semester', 'Building', 'Room', 'Times', '', ''];
+
+    const [search, setSearch] = useState({
+        scheduleId: '',
+        year: '',
+        semester: ''
+    });
+    const [message, setMessage] = useState('');
+    const [schedules, setSchedules] = useState([]);
+
+    const fetchSections = async () => {
+            if (search.courseId==='' || search.year==='' || search.semester==='' ) {
+                setMessage("Enter search parameters");
+            } else {
+              try {
+                const response = await fetch(`${SERVER_URL}/courses/${search.courseId}/sections?year=${search.year}&semester=${search.semester}`);
+                if (response.ok) {
+                  const data = await response.json();
+                  setSchedules(data);
+                } else {
+                  const rc = await response.json();
+                  setMessage(rc.message);
+                }
+              } catch(err) {
+                setMessage("network error: "+err);
+              }
+            }
+        }
+
+        const deleteSchedule = async (secNo) => {
+          try {
+            const response = await fetch (`${SERVER_URL}/sections/${secNo}`,
+            {
+              method: 'DELETE',
+              headers: {
+                'Content-Type': 'application/json',
+              },
+            });
+            if (response.ok) {
+              setMessage("Schedule deleted");
+              fetchSections();
+            } else {
+              const rc = await response.json();
+              setMessage(rc.message);
+            }
+          } catch (err) {
+            setMessage("network error: "+err);
+          }
+        }
+
+
+        const editChange = (event) => {
+            setSearch({...search,  [event.target.name]:event.target.value});
```

Comment:
If we are populating the DB ourselves it would be nice to make everything lower case so that we could make searching easier here. we could toLowerCase() on the input before setting `search`. at least for course and semester.